--- openmovieeditor-0.0.20071118/configure.ac.orig	2007-12-18 20:55:48.402203456 +0100
+++ openmovieeditor-0.0.20071118/configure.ac	2007-12-18 20:58:25.378255776 +0100
@@ -53,29 +53,29 @@
 PKG_CHECK_MODULES(SAMPLERATE, samplerate >= 0.1.1,,
   AC_MSG_ERROR([*** libsamplerate not found - this program requires libsamplerate >= 0.1.1 http://www.mega-nerd.com/SRC/ ***]))
 CPPFLAGS="$CPPFLAGS $SAMPLERATE_CFLAGS"
-LDFLAGS="$LDFLAGS $SAMPLERATE_LIBS"
+LIBS="$LIBS $SAMPLERATE_LIBS"
 
 PKG_CHECK_MODULES(SNDFILE, sndfile >= 1.0.0,,
   AC_MSG_ERROR([*** libsndfile 1.0.0 not found - this program requires libsndfile >= 1.0.0 http://www.mega-nerd.com/libsndfile/ ***]))
 CPPFLAGS="$CPPFLAGS $SNDFILE_CFLAGS"
-LDFLAGS="$LDFLAGS $SNDFILE_LIBS"
+LIBS="$LIBS $SNDFILE_LIBS"
 
 PKG_CHECK_MODULES(GAVL, gavl >= 1.0.0,,
   AC_MSG_ERROR([*** gavl >= 1.0.0 not found - this program requires the gmerlin audio
   video library >= 1.0.0 http://gmerlin.sourceforge.net/ ***]))
 CPPFLAGS="$CPPFLAGS $GAVL_CFLAGS"
-LDFLAGS="$LDFLAGS $GAVL_LIBS"
+LIBS="$LIBS $GAVL_LIBS"
 
 #PKG_CHECK_MODULES(GMERLIN_AVDEC, gmerlin_avdec >= 0.1.7,,
 #  AC_MSG_ERROR([*** gmerlin_avdec >= 0.1.7 not found - this program requires the gmerlin audio
 #  video decoder >= 0.1.7 http://gmerlin.sourceforge.net/ ***]))
 #CPPFLAGS="$CPPFLAGS $GMERLIN_AVDEC_CFLAGS"
-#LDFLAGS="$LDFLAGS $GMERLIN_AVDEC_LIBS"
+#LIBS="$LIBS $GMERLIN_AVDEC_LIBS"
 
 PKG_CHECK_MODULES(LIBQUICKTIME, libquicktime >= 1.0.2,,
   AC_MSG_ERROR([*** libquicktime 1.0.2 not found - this program requires libquicktime >= 1.0.2 http://libquicktime.sourceforge.net/ ***]))
 CPPFLAGS="$CPPFLAGS $LIBQUICKTIME_CFLAGS"
-LDFLAGS="$LDFLAGS $LIBQUICKTIME_LIBS"
+LIBS="$LIBS $LIBQUICKTIME_LIBS"
 
 HAVE_AVCODEC=false
 PKG_CHECK_MODULES(AVCODEC, libavcodec, [HAVE_AVCODEC=true], [true])
@@ -83,7 +83,7 @@
   AC_MSG_WARN([*** libavcodec not found])
 else
   CPPFLAGS="$CPPFLAGS $AVCODEC_CFLAGS -DAVCODEC"
-  LDFLAGS="$LDFLAGS $AVCODEC_LIBS"
+  LIBS="$LIBS $AVCODEC_LIBS"
 fi
 
 if test x$HAVE_AVCODEC = xfalse; then
@@ -93,7 +93,7 @@
     AC_MSG_WARN([*** libavcodec_acl not found])
   else
     CPPFLAGS="$CPPFLAGS $AVCODEC_ACL_CFLAGS -DAVCODEC"
-    LDFLAGS="$LDFLAGS $AVCODEC_ACL_LIBS"
+    LIBS="$LIBS $AVCODEC_ACL_LIBS"
   fi
 fi
 
@@ -103,7 +103,7 @@
   AC_MSG_WARN([*** libavformat not found])
 else
   CPPFLAGS="$CPPFLAGS $AVFORMAT_CFLAGS"
-  LDFLAGS="$LDFLAGS $AVFORMAT_LIBS"
+  LIBS="$LIBS $AVFORMAT_LIBS"
 fi
 
 if test x$HAVE_AVFORMAT = xfalse; then
@@ -113,7 +113,7 @@
     AC_MSG_WARN([*** libavformat_acl not found])
   else
     CPPFLAGS="$CPPFLAGS $AVFORMAT_ACL_CFLAGS"
-    LDFLAGS="$LDFLAGS $AVFORMAT_ACL_LIBS"
+    LIBS="$LIBS $AVFORMAT_ACL_LIBS"
   fi
 fi
 
@@ -129,13 +129,13 @@
   AC_MSG_WARN([*** libswscale not found])
 else
   CPPFLAGS="$CPPFLAGS $SWSCALE_CFLAGS -DSWSCALE"
-  LDFLAGS="$LDFLAGS $SWSCALE_LIBS"
+  LIBS="$LIBS $SWSCALE_LIBS"
 fi
 
 PKG_CHECK_MODULES(JACK, jack, ac_jack_lib=yes, ac_jack_lib=no)
 AC_CHECK_HEADER(jack/jack.h, [ac_jack_h="yes"], [ac_jack_h="no"])
 if test "x$ac_jack_lib$ac_jack_h" = "xyesyes"; then
-   LDFLAGS="$LDFLAGS -ljack"
+   LIBS="$LIBS -ljack"
 else 
    AC_MSG_ERROR([*** You need JACK audio library (libjack-dev) installed.  http://jackit.sf.net/])
 fi
@@ -143,12 +143,12 @@
 PKG_CHECK_MODULES(GLIB2, glib-2.0 >= 2.10.3,,
   AC_MSG_ERROR([*** glib-2.0 not found - this program requires glib >= 2.10.3 ***]))
 CPPFLAGS="$CPPFLAGS $GLIB2_CFLAGS"
-LDFLAGS="$LDFLAGS $GLIB2_LIBS"
+LIBS="$LIBS $GLIB2_LIBS"
 
 PKG_CHECK_MODULES(PORTAUDIO19, portaudio-2.0 >= 19,,
   AC_MSG_ERROR([*** portaudio-2.0 not found - this program requires portaudio >= 19 ***]))
 CPPFLAGS="$CPPFLAGS $PORTAUDIO19_CFLAGS"
-LDFLAGS="$LDFLAGS $PORTAUDIO19_LIBS"
+LIBS="$LIBS $PORTAUDIO19_LIBS"
 
  AC_CHECK_HEADER(GL/gl.h,,
   AC_MSG_ERROR([*** GL/gl.h not found - this program requires OpenGL or compatible (Mesa) Hint: install mesa-common-dev ***]))
@@ -185,7 +185,7 @@
 dnl  fi
 fi
 CPPFLAGS="$CPPFLAGS $FLTK_CXXFLAGS"
-LDFLAGS="$LDFLAGS $FLTK_LIBS"
+LIBS="$LIBS $FLTK_LIBS"
 AC_SUBST(FLTK_CXXFLAGS)
 AC_SUBST(FLTK_CFLAGS)
 AC_SUBST(FLTK_LIBS)
