--- openmovieeditor-0.0.20071118/configure.ac.orig	2007-12-18 20:55:48.402203456 +0100
+++ openmovieeditor-0.0.20071118/configure.ac	2007-12-18 20:58:25.378255776 +0100
@@ -61,23 +61,23 @@
 PKG_CHECK_MODULES(SAMPLERATE, samplerate >= 0.1.1,,
   AC_MSG_ERROR([*** libsamplerate not found - this program requires libsamplerate >= 0.1.1 http://www.mega-nerd.com/SRC/ ***]))
 CPPFLAGS="$CPPFLAGS $SAMPLERATE_CFLAGS"
-LDFLAGS="$LDFLAGS $SAMPLERATE_LIBS"
+LIBS="$LIBS $SAMPLERATE_LIBS"
 
 PKG_CHECK_MODULES(SNDFILE, sndfile >= 1.0.0,,
   AC_MSG_ERROR([*** libsndfile 1.0.0 not found - this program requires libsndfile >= 1.0.0 http://www.mega-nerd.com/libsndfile/ ***]))
 CPPFLAGS="$CPPFLAGS $SNDFILE_CFLAGS"
-LDFLAGS="$LDFLAGS $SNDFILE_LIBS"
+LIBS="$LIBS $SNDFILE_LIBS"
 
 PKG_CHECK_MODULES(GAVL, gavl >= 0.2.3,,
   AC_MSG_ERROR([*** gavl >= 0.2.3 not found - this program requires the gmerlin audio
   video library >= 0.2.3 http://gmerlin.sourceforge.net/ ***]))
 CPPFLAGS="$CPPFLAGS $GAVL_CFLAGS"
-LDFLAGS="$LDFLAGS $GAVL_LIBS"
+LIBS="$LIBS $GAVL_LIBS"
 
 PKG_CHECK_MODULES(LIBQUICKTIME, libquicktime >= 0.9.7,,
   AC_MSG_ERROR([*** libquicktime 0.9.7 not found - this program requires libquicktime >= 0.9.7 http://libquicktime.sourceforge.net/ ***]))
 CPPFLAGS="$CPPFLAGS $LIBQUICKTIME_CFLAGS"
-LDFLAGS="$LDFLAGS $LIBQUICKTIME_LIBS"
+LIBS="$LIBS $LIBQUICKTIME_LIBS"
 
 HAVE_AVCODEC=false
 PKG_CHECK_MODULES(AVCODEC, libavcodec, [HAVE_AVCODEC=true], [true])
@@ -80,7 +80,7 @@
   AC_MSG_WARN([*** libavcodec not found])
 else
   CPPFLAGS="$CPPFLAGS $AVCODEC_CFLAGS -DAVCODEC"
-  LDFLAGS="$LDFLAGS $AVCODEC_LIBS"
+  LIBS="$LIBS $AVCODEC_LIBS"
 fi
 
 if test x$HAVE_AVCODEC = xfalse; then
@@ -90,7 +90,7 @@
     AC_MSG_WARN([*** libavcodec_acl not found])
   else
     CPPFLAGS="$CPPFLAGS $AVCODEC_ACL_CFLAGS -DAVCODEC"
-    LDFLAGS="$LDFLAGS $AVCODEC_ACL_LIBS"
+    LIBS="$LIBS $AVCODEC_ACL_LIBS"
   fi
 fi
 
@@ -100,7 +100,7 @@
   AC_MSG_WARN([*** libavformat not found])
 else
   CPPFLAGS="$CPPFLAGS $AVFORMAT_CFLAGS"
-  LDFLAGS="$LDFLAGS $AVFORMAT_LIBS"
+  LIBS="$LIBS $AVFORMAT_LIBS"
 fi
 
 if test x$HAVE_AVFORMAT = xfalse; then
@@ -110,7 +110,7 @@
     AC_MSG_WARN([*** libavformat_acl not found])
   else
     CPPFLAGS="$CPPFLAGS $AVFORMAT_ACL_CFLAGS"
-    LDFLAGS="$LDFLAGS $AVFORMAT_ACL_LIBS"
+    LIBS="$LIBS $AVFORMAT_ACL_LIBS"
   fi
 fi
 
@@ -118,7 +118,7 @@
 AC_CHECK_LIB(jack, main, [ac_jack_lib="yes"], [ac_jack_lib="no"])
 AC_CHECK_HEADER(jack/jack.h, [ac_jack_h="yes"], [ac_jack_h="no"])
 if test "x$ac_jack_lib$ac_jack_h" = "xyesyes"; then
-   LDFLAGS="$LDFLAGS -ljack"
+   LIBS="$LIBS -ljack"
 else 
    AC_MSG_ERROR([*** You need JACK audio library (libjack-dev) installed.  http://jackit.sf.net/])
 fi
@@ -155,7 +155,7 @@
 dnl  fi
 fi
 CPPFLAGS="$CPPFLAGS $FLTK_CXXFLAGS"
-LDFLAGS="$LDFLAGS $FLTK_LIBS"
+LIBS="$LIBS $FLTK_LIBS"
 AC_SUBST(FLTK_CXXFLAGS)
 AC_SUBST(FLTK_CFLAGS)
 AC_SUBST(FLTK_LIBS)
