--- configure.ac	2006-07-12 21:48:19.000000000 +0200
+++ configure.ac	2006-07-12 21:48:06.000000000 +0200
@@ -55,9 +55,9 @@
 CPPFLAGS="$CPPFLAGS $SNDFILE_CFLAGS"
 LDFLAGS="$LDFLAGS $SNDFILE_LIBS"
 
-PKG_CHECK_MODULES(GAVL, gavl = 0.2.3,,
-  AC_MSG_ERROR([*** gavl 0.2.3 not found - this program requires the gmerlin audio
-  video library = 0.2.3 http://gmerlin.sourceforge.net/ ***]))
+PKG_CHECK_MODULES(GAVL, gavl = 0.2.4,,
+  AC_MSG_ERROR([*** gavl 0.2.4 not found - this program requires the gmerlin audio
+  video library = 0.2.4 http://gmerlin.sourceforge.net/ ***]))
 CPPFLAGS="$CPPFLAGS $GAVL_CFLAGS"
 LDFLAGS="$LDFLAGS $GAVL_LIBS"
 
